cmd/rabbitmq-setup/main.go

```diff
@@ -54,7 +54,7 @@ func main() {
 
 	_, err = ch.QueueDeclare(
 		monitorQueueName,
-		amqpDurable,
```

Comment:
amqpExchangeDurable vs amqpQueueDurable?

I know this is just a hack around the not great API but it's a hack that's kind of okay.